pkg/proxy/proxyports/netstat.go

```diff
@@ -9,9 +9,12 @@ import (
 	"regexp"
 	"strconv"
 
+	"github.com/cilium/cilium/pkg/logging"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 )
 
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "proxy")
+
```

Comment:
This seems to be removed in subsequent commit

Reply:
yep, this was while being in process (never 100% sure whether you are able to remove all usages in the same PR - therefore moving it closer to the usages that are left) :smile: